src/JuMP.jl

```diff
@@ -298,7 +306,7 @@ end
 """
     mode(model::Model)
 
-Return mode (DIRECT, AUTOMATIC, MANUAL) of model.
+Return the `ModelMode` (`DIRECT`, `AUTOMATIC`, or `MANUAL`) of `model`.
```

Comment:
I see that `ModelMode` has a docstring. Shouldn't it be in the doc so that this can reference it with `[@ref]` ?